settings.py

```diff
@@ -124,6 +124,8 @@
 # Validation key we need to send in POST response.
 INBOUND_EMAIL_VALIDATION_KEY = 'totally-unsecure-validation-string'
 
+CODE_MANAGER_URL = 'http://localhost:3000'
```

Comment:
Isn't this the same port that addons-frontend would run on?  You can't run both at once?

Reply:
That's true, it is the same port and you can't run both. Honestly, I don't think it would be uncommon to hook up a local `addons-server` to a local `addons-code-manager` so this is probably fine just as a reference. I could change it to the dev URL but I'm not sure if that would be useful since the data won't match (URLs will be 404s).

Reply:
I thought you'd be running it on a different port by default anyway for that reason.  But I suppose it's an edge case you'd need that link to work reliably.